packages/application-generic/src/services/cache/key-builders/queries.ts

```diff
@@ -134,38 +138,40 @@ export const buildQueryKey = ({
     identifier,
   })}:${QUERY_PREFIX}=${JSON.stringify(query)}`;
 
-export const buildQueryByEnvironmentKey = ({
+export const buildQueryByOrganizationKey = ({
   type,
   keyEntity,
-  environmentIdPrefix = OrgScopePrefixEnum.ENVIRONMENT_ID,
-  environmentId,
+  organizationIdPrefix = OrgScopePrefixEnum.ORGANIZATION_ID,
+  organizationId,
   query,
 }: {
   type: CacheKeyTypeEnum;
   keyEntity: CacheKeyPrefixEnum;
-  environmentIdPrefix?: OrgScopePrefixEnum;
-  environmentId: string;
+  organizationIdPrefix?: OrgScopePrefixEnum;
+  organizationId: string;
   query: Record<string, unknown>;
 }): string =>
-  `${buildKeyByEnvironment({
+  `${buildKeyByOrganization({
     type,
     keyEntity,
-    environmentIdPrefix,
-    environmentId,
+    organizationIdPrefix,
+    organizationId,
   })}:${QUERY_PREFIX}=${JSON.stringify(query)}`;
```

Comment:
Here we can see that we specify the cache key where the key changed from environment to organization and that the environment is now part of the query key.